datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py

```diff
@@ -804,7 +804,19 @@ def _submit_gauges_from_summary(self, metric_name, metric, scraper_config, hostn
                     hostname=custom_hostname,
                 )
             else:
-                sample[self.SAMPLE_LABELS]["quantile"] = str(float(sample[self.SAMPLE_LABELS]["quantile"]))
+                try:
+                    quantile = sample[self.SAMPLE_LABELS]["quantile"]
+                except KeyError:
+                    # In the Prometheus spec the 'quantile' label is optional, but it's not clear yet
+                    # what we should do in this case. Let's skip for now and submit the rest of metrics.
+                    message = (
+                        '"quantile" label not present in metric %r.'
+                        'Quantile-less summary metrics are not currently supported. Skipping...'
+                    )
+                    self.log.warning(message, metric_name)
```

Comment:
Should we use `debug` instead ? This might spam user and he won't be able to do anything about it.